src/pudl/transform/params/ferc1.py

```diff
@@ -2623,7 +2623,93 @@
                     "adjustments": "adjustments",
                     "transfers": "transfers",
                     "ending_balance": "ending_balance",
-                }
+                },
+                "not_renamed_columns": [
```

Comment:
Lol okay it does necessitate a bunch of new metadata.

I assume that you've programmatically generated these lists of not renamed columns. Are they just all of the columns that weren't getting renamed? If so, do we need to worry that we might be codifying errors in specifying them now? Like, are any of these columns that we weren't renaming actually columns that we *should* have been renaming, but weren't resulting in their contents getting lost when we eventually `enforce_schema()` at the end of the transform step?

Reply:
Yes, these are all generated programatically. I might suggest sorting the columns in the error message, to make it easier to spot near-duplicates/typos.

However, the overwhelming majority of these columns are factoid names, which we're collapsing into the `xbrl_factoid` column, so they wouldn't be affected by `enforce_schema`. Several others are not and should probably be checked.